pyproject.toml

```diff
@@ -12,7 +12,7 @@ dependencies = [
     "sqlalchemy>=2.0.0",
     "uvicorn[standard]>=0.24.0",
     "websockets>=12.0",
-    "libtmux>=0.37.0",
+    "libtmux>=0.51.0",
```

Comment:
Can we lock the dependency version with `=` instead of `>=`? No need to keep doing this for backward incompatible changes.